machine_learning/single_neuron/single_neuron.py

```diff
@@ -0,0 +1,40 @@
+import numpy as np
+from sklearn import datasets
+
+epochs = 2000
+digits = datasets.load_breast_cancer()
+X, y = digits.data, digits.target
+
+b = len(X)
+z = np.array([[1] * b]).T
+X = np.concatenate([z, X], axis = 1)
+c = len(X[0])
+del(z)
+
+w = np.array([np.random.random()]*c)
+
+def relu_func(x):
+	if x > 0:
+		return 1
```

Comment:
correct me if I'm wrong but shouldn't it be

if x > 0:
   return x
else:
   return 0

or 

max(0,x)

Reply:
You are right, what you described is really relu I was doing classification so I needed to convert the value in discrete variable that's why I did that. I converted that variable into 0 and 1 which are also the target variables.